docs/UserGuide.md

```diff
@@ -1,48 +1,49 @@
----
-layout: page
-title: User Guide
----
-
 EventfulNUS is a **desktop app for managing contacts and events specifically for the Inter-Faculty Games hosted annually
 at the National University of Singapore. While optimised for use via a Command Line Interface** (CLI), it also has the benefits of a Graphical User Interface (GUI). If you can type fast, you will certainly benefit from event organisation tasks being done faster than traditional GUI apps.
 
 ---------------------------------------------------------------------------------------------------------------------
 ## Table of Contents
 1. [Quick start](#quick-start)
 2. [Features](#features)
-3. [Person commands](#persons)
-4. [Event commands](#events)
-5. [More features](#more-features)
-6. [FAQ](#faq)
-7. [Known issues](#known-issues)
-8. [Command summary](#command-summary)
+   * [Person commands](#persons)
+   * [Event commands](#events)
+   * [Miscellaneous features](#miscellaneous-features)
+3. [FAQ](#faq)
+4. [Known issues](#known-issues)
+5. [Command summary](#command-summary)
 --------------------------------------------------------------------------------------------------------------------
 ## Quick start
 
 1. Ensure you have Java `17` or above installed in your Computer.
 
-1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
+2. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
 
-1. Copy the file to the folder you want to use as the _home folder_ for EventfulNUS.
+3. Copy the file to the folder you want to use as the _home folder_ for EventfulNUS.
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar tp.jar` command to run the application.<br>
+4. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar tp.jar` command to run the application.<br>
```

Comment:
The non-tech audience does not know what 'cd' into the folder means. Can we give them a step by step onto how to open the terminal and change directories into the folder they saved the jar file?

Reply:
can you give me an idea of how to write such a guide?

Reply:
I've made a commit for this, please let me know what you think!

Reply:
looks good